backend/api/views/message.py

```diff
@@ -108,6 +108,12 @@ def add_message():
         f"Your Field Partner from {organization} has uploaded a document for {docclass_name}.",  # organization, document class name
     ]
 
+    if "reason" in data:
+        reason = data.get("reason")
+        contents[
+            1
```

Comment:
Does this actually look like this in the code bc if so this is weirdest post-lint format thing I've seen